relay-common/src/constants.rs

```diff
@@ -120,6 +120,8 @@ pub enum DataCategory {
     /// This is the category for transaction payloads that were accepted and stored in full. In
     /// contrast, `transaction` only guarantees that metrics have been accepted for the transaction.
     TransactionIndexed = 9,
+    /// Cron monitor checkins.
+    Cron = 10,
```

Comment:
I wonder if we should call this monitor, we are somewhat consistently calling crons `monitors` within the codebase, and `crons` as the product name at the moment.

Reply:
I would also rename it to the `CronMonitor`
```suggestion
    CronMonitor = 10,
```

Reply:
It's worth noting in the future these may. also support heart beat checkins, at which point the word "cron" may be a bit confusing. but idk. my preference is on keeping this consistent with what we have in code. I don't think we say `CronMonitor` anywhere in the api yet.

Reply:
I'll just go with "Monitor" everywhere to keep it concise.